package/harvester-os/files/system/oem/91_installer.yaml

```diff
@@ -3,6 +3,7 @@ stages:
   boot:
     - commands:
       - setup-installer.sh
+      if: '[ ! -n "$(grep "harvester.installer.disable" /proc/cmdline)" ] || ! grep -q "harvester.installer.disable=true" /proc/cmdline'
```

Comment:
Seems it can achieve the same results with only the second part. Is there any concern to leave `[ ! -n "$(grep "harvester.installer.disable" /proc/cmdline)" ]` here? Thanks!

Reply:
Nice catch! Because I changed the if condition, the first part could be dropped!

Reply:
Updated! @starbops, please recheck it.